opentelemetry-appender-log/Cargo.toml

```diff
@@ -12,12 +12,16 @@ edition = "2021"
 
 [dependencies]
 opentelemetry = { version = "0.22", path = "../opentelemetry", features = ["logs"]}
-log = { workspace = true, features = ["kv_unstable", "std"]}
+log = { workspace = true, features = ["kv", "std"]}
+serde = { workspace = true, optional = true, features = ["std"] }
 
 [features]
 logs_level_enabled = ["opentelemetry/logs_level_enabled"]
+with_serde = ["log/kv_serde", "serde"]
```

Comment:
At other places, we use `with-serde` as feature name. Is `with_serde` naming intentional?

Reply:
I just tried to follow the convention I saw in naming of other features. I can change this to kebab-case 👍